samza-core/src/main/scala/org/apache/samza/util/FileUtil.scala

```diff
@@ -26,9 +26,13 @@ import java.nio.file._
 import java.util.zip.CRC32
 
 class FileUtil extends Logging {
+  val VERSION = 1
+  val MAX_WRITE_UTF_SEGMENT = 0xFFFF
```

Comment:
Prefer a human readable val name and value. E.g. `val MaxStringSizeInBytes = 64 * 1024`.

Also, scala convention for statics is PascalCase: https://docs.scala-lang.org/style/naming-conventions.html#constants-values-and-variables

Reply:
Changed thanks